From 1601be1a3a3c758cdc4c5a0ecdcf65b1e812e301 Mon Sep 17 00:00:00 2001
From: Thomas Tuegel <ttuegel@mailbox.org>
Date: Wed, 18 Oct 2017 08:15:49 -0500
Subject: [PATCH 02/11] Fix binary install path in D-Bus service files

Some D-Bus service files assume
CMAKE_INSTALL_FULL_BINDIR == CMAKE_INSTALL_PREFIX/bin
which is certainly not true in Nixpkgs.
---
 krunner/dbus/org.kde.krunner.service.in                      | 2 +-
 kuiserver/org.kde.kuiserver.service.in                       | 2 +-
 startkde/waitforname/org.kde.plasma.Notifications.service.in | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/krunner/dbus/org.kde.krunner.service.in b/krunner/dbus/org.kde.krunner.service.in
index 85715214..294eab08 100644
--- a/krunner/dbus/org.kde.krunner.service.in
+++ b/krunner/dbus/org.kde.krunner.service.in
@@ -1,4 +1,4 @@
 [D-BUS Service]
 Name=org.kde.krunner
-Exec=@CMAKE_INSTALL_PREFIX@/bin/krunner
+Exec=@CMAKE_INSTALL_FULL_BINDIR@/krunner
 
diff --git a/kuiserver/org.kde.kuiserver.service.in b/kuiserver/org.kde.kuiserver.service.in
index 7a86d07f..5b3030cc 100644
--- a/kuiserver/org.kde.kuiserver.service.in
+++ b/kuiserver/org.kde.kuiserver.service.in
@@ -1,3 +1,3 @@
 [D-BUS Service]
 Name=org.kde.kuiserver
-Exec=@CMAKE_INSTALL_PREFIX@/bin/kuiserver5
+Exec=@CMAKE_INSTALL_FULL_BINDIR@/kuiserver5
diff --git a/startkde/waitforname/org.kde.plasma.Notifications.service.in b/startkde/waitforname/org.kde.plasma.Notifications.service.in
index 0a51b84b..f48b5d8a 100644
--- a/startkde/waitforname/org.kde.plasma.Notifications.service.in
+++ b/startkde/waitforname/org.kde.plasma.Notifications.service.in
@@ -1,3 +1,3 @@
 [D-BUS Service]
 Name=org.freedesktop.Notifications
-Exec=@CMAKE_INSTALL_PREFIX@/bin/plasma_waitforname org.freedesktop.Notifications
+Exec=@CMAKE_INSTALL_FULL_BINDIR@/plasma_waitforname org.freedesktop.Notifications
-- 
2.16.1

